packages/webdriverio/tests/utils/Timer.test.js

```diff
@@ -0,0 +1,57 @@
+import Timer from '../../src/utils/Timer'
+// import sinon from 'sinon'
+import assert from 'assert'
```

Comment:
Let's use jest's assertion library throughout the tests

Reply:
Forgot to remove this, it wasn't even being used